src/pages/studyView/addChartButton/customCaseSelection/CustomCaseSelectionUtils.tsx

```diff
@@ -44,6 +51,11 @@ export type ValidationMessage = {
     message: Error
 };
 
+export type InformationMessage = {
```

Comment:
I don't see much benefits adding a separate `InformationMessage ` here. When `ALL_NUMERIC ` listed under validation result, then we know it's ALL_NUMERICAL. We could still use ValidationMessage and make the message to Error | String